### PR TITLE
Cleanup register-device getting started page

### PR DESCRIPTION
**Pull request by kprosise**
Fixed trademark and heading issues. Changed markdown to use semantic newlines. Reduced wordiness.

QA Steps: Ran linkcheck. Final run an issue on a different page with an external link showed up, this is an external issue and can be ignored. Rendered and checked html output for correct visual formatting. Linted.

No Jira issues to tag, routine cleanup.

Signed-off-by: Katrina Prosise <katrina.prosise@foundries.io>

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

Routine cleanup.

## Checklist

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for b13bf9d19b066050b1019549027974a2a787a9d4 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1915/docs/artifacts/html/index.html

File: source/getting-started/register-device/index.rst [hunk 1]
```diff
@@ -1,27 +1,23 @@
...
-	the :term:`containers.git` repository and therefore available in the
-	latest Target. This behavior can be changed by enabling only specific
-	applications. Read :ref:`ug-fioctl-enable-apps` to learn how.
+    **By default**, devices will run **all** applications that are defined in the :term:`containers.git` repository,
+    thus available in the latest Target. This behavior can be changed by enabling only specific applications.
+    Read :ref:`ug-fioctl-enable-apps` to learn how.
 
-2. You will be prompted by ``lmp-device-register`` to complete a challenge with
-   our API
+2. You will be prompted by ``lmp-device-register`` to complete a challenge with our API:
```
Comment (doanac) [inline source/getting-started/register-device/index.rst:20]: Not related to this change exactly, but people sometimes ask about this challenge. We use Oauth2 Device Flow - https://www.oauth.com/oauth2-servers/device-flow/. maybe we link to that here.

**Review by doanac — APPROVED** [review]
looks good.

File: source/getting-started/register-device/index.rst (hunk 1 above)
Comment (kprosise) [inline source/getting-started/register-device/index.rst:20]:
```suggestion
2. You will be prompted by ``lmp-device-register`` to `complete a challenge <https://www.oauth.com/oauth2-servers/device-flow/>`_ with our API:
```
@doanac, does this work?

**Review by kprosise — COMMENTED** [review]
suggestion adding link to more info related to the challenge

File: source/getting-started/register-device/index.rst (hunk 1 above)
Comment (doanac) [reply source/getting-started/register-device/index.rst:20]: yep. thanks!

**Comment by doanac** [comment]
Docs for 89c6c2ab67461a30acb11ab8839ce546aea01f98 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1918/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 2b9aa394a7a79cb49f75324b508ae6235fbfe040 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1919/docs/artifacts/html/index.html

**Pull request merged**